storm-core/src/clj/org/apache/storm/testing.clj

```diff
@@ -288,11 +289,11 @@
 (defmacro with-inprocess-zookeeper
   [port-sym & body]
   `(with-local-tmp [tmp#]
-                   (let [[~port-sym zks#] (zk/mk-inprocess-zookeeper tmp#)]
+                   (let [[~port-sym zks#] (Zookeeper/mkInprocessZookeeper tmp# nil)]
```

Comment:
It didn't I fixed it in a different way.  I made Zookeeper/mkInprocessZookeeper match zk/mk-inprocess-zookeeper instead of returning just the factory and trying to pull the port out of the factory.